meshtastic/protobuf/config_pb2.pyi

```diff
@@ -56,6 +56,7 @@ class Config(google.protobuf.message.Message):
             ROUTER_CLIENT: Config.DeviceConfig._Role.ValueType  # 3
             """
             Description: Combination of both ROUTER and CLIENT. Not for mobile devices.
+            Deprecated in v2.3.15 because improper usage is impacting public meshes: Use ROUTER or CLIENT instead.
```

Comment:
Presumably you needed the v2.3.15 protobufs for this, given that's where the powermon stuff gets merged? I suppose I'll use 2.3.14 as a final-version-that-supports-3.8, I think, before merging this, and then I can just pull in protobufs v2.3.15 outright.

Reply:
no idea - that string came from meshtastic/protobufs and I just merged it with my changes.